cpp/src/gandiva/regex_functions_holder_test.cc

```diff
@@ -46,6 +46,7 @@ class TestLikeHolder : public ::testing::Test {
 };
 
 TEST_F(TestLikeHolder, TestMatchAny) {
+  regex_op.set_dot_nl(true);
```

Comment:
Can this set_dot_nl be moved to the constructor, i.e. enable it for all test cases?